### ISSUE TITLE
Change the url/title separator from `-` to ` | ` with spaces

### ISSUE THREAD

**Issue by ericpromislow** [ericpromislow/tabhunter#125]
When a URL ends with a hyphenated phrase, it's not obvious where the URL ends and the title begins.

**Comment by swadeley** [comment]
Hi, this has always been troublesome for me when copying the URL and title. 

For example, this is what I get for this page :
`https://github.com/ericpromislow/tabhunter/issues/125-Change the url/title separator from `-` to ` | ` with spaces · Issue #125 · ericpromislow/tabhunter`

Thank you

**Comment by ericpromislow** [comment]
Fixed in commit 1c261b8

**Comment by swadeley** [comment]
Hi

This is great, thank you:
```
https://github.com/ericpromislow/tabhunter/issues/125 | Change the url/title separator from `-` to ` | ` with spaces · Issue #125 · ericpromislow/tabhunter
```
